src/containers/NFTView/NFTView.tsx

```diff
@@ -231,7 +231,9 @@ const NFTView: React.FC<NFTViewProps> = ({ domain, onTransfer }) => {
 
 	const history = () => (
 		<section
-			className={`${styles.History} ${styles.Box} blur border-primary border-rounded`}
+			className={`${styles.History} ${
+				bids && bids.length === 0 ? styles.HistoryNoBids : ' '
```

Comment:
You can write the conditional here as `bids?.length === 0 ?`, saves a lil bit of space. Obviously not needed for approval, just letting you know that's a thing. It's called [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
Cool. Sometimes I do what you've suggested but typescript gets funny about it sometimes. Thank you.